<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpanDialog</class>
 <widget class="QWidget" name="SpanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Span/Interval</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="TimeStartF">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Time Start (GPST)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="TimeEndF">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Time End (GPST)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="TimeIntF">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Interval (s)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="BtnOk">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="EditTimeInt">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>0.1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>60</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>300</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3600</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="BtnCancel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDateTimeEdit" name="dateTime1">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDateTimeEdit" name="dateTime2">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
